README.md

```diff
@@ -21,77 +21,19 @@ BandChain - Decentralized Data Delivery Network<br/><br/>
-
-We recommend the following for running a BandChain Validator:
-
-- **2 or more** CPU cores
-- **8 GB** of RAM (16 GB in case on participate in mainnet upgrade)
-- At least **100GB** of disk storage
-
-**Step 1. Install Golang**
-
-Go v1.18+ or higher is required for BandChain.
-
-If you haven't already, install Golang by following the [official docs](https://golang.org/doc/install). Make sure that your GOPATH and GOBIN environment variables are properly set up.
-
-**Step 2. Get BandChain source code**
-
-Use `git` to retrieve BandChain from the [official repo](https://github.com/bandprotocol/chain), and checkout the master branch, which contains the latest stable release. That should install the `bandd` binary.
-
-```bash
-git clone https://github.com/bandprotocol/chain
-cd chain && git checkout v2.3.0
-make install
-```
-
-**Step 3. Verify your installation**
-
-Using `bandd version` command to verify that your `bandd` has been build successfully.
-
-```
-bandd version --long
-name: bandchain
-server_name: bandd
-version: 2.3.0
-commit: 4fe19638b33043eed4dec9861cda40962fb5b2a7
-build_tags: ledger
-go: go version go1.18.3 darwin/amd64
-build_deps:
-...
-```
-
-### Setting Up Yoda — The Oracle Daemon
-
-BandChain validators are also responsible for responding to oracle data requests. Whenever someone submits a request message to BandChain, the chain will autonomously choose a subset of active oracle validators to perform the data query.
-
-The validators are chosen submit a report message to BandChain within a given timeframe as specified by a chain parameter (100 blocks in mainnet). We provide a program called yoda to do this task for you. For more information on the data request process, please see [here](https://docs.bandchain.org/whitepaper/system-overview.html#oracle-data-request).
-
-Yoda uses an external executor to resolve requests to data sources. Currently, it supports [AWS Lambda](https://aws.amazon.com/lambda/) and [Google Cloud Function](https://cloud.google.com/functions) (through the REST interface). In future releases, `yoda` will support more executors and allow you to specify multiple executors to add redundancy.
-
-You also need to set up `yoda` and activate oracle status. Here’s the [documentation](https://github.com/bandprotocol/bandchain/wiki/Instruction-for-apply-to-be-an-oracle-validator-on-Guanyu-mainnet) to get started.
-
-That’s it! You can verify that your validator is now an oracle provider via cli by using ` bandd query oracle validator <your validator address>`. Your yoda process must be responding to oracle requests assigned to your node. If the process misses a request, your oracle provider status will automatically get deactivated and you must send MsgActivate to activate again after a 10-minute waiting period and make sure that yoda is up.
+Please refer to [this documentation](https://docs.bandchain.org/node-validators/run-node/joining-mainnet/installation) for the most up-to-date installation guide.
```

Comment:
as discussed, separate building from source from installation as it's versioning.